### ISSUE TITLE
[Feature Request] Prop to show a "v-text-field like" v-date-picker 

### ISSUE THREAD

**Issue by simionato-mobisec** [vuetifyjs/vuetify#18184]
### Problem to solve
This feature would allow to create more webapp like forms.

### Proposed solution
https://primevue.org/calendar/#time

**Comment by jchown** [comment]
You can use `v-date-picker` with `input-mode='keyboard'`?

**Comment by simionato-mobisec** [comment]
> You can use `v-date-picker` with `input-mode='keyboard'`?

Yes I can, but the result won't be like the one in the example I attached.
